backend/vaa-strapi/docker-compose.dev.yml

```diff
@@ -15,7 +15,7 @@ services:
       API_TOKEN_SALT: ${API_TOKEN_SALT}
       ADMIN_JWT_SECRET: ${ADMIN_JWT_SECRET}
       JWT_SECRET: ${JWT_SECRET}
-      DATABASE_HOST: ${DATABASE_HOST}
+      DATABASE_HOST: postgres
```

Comment:
The reason being that within the compose set the DB host matches the PostgresDB service name and always stays the same.